src/webhook.py

```
@@ -0,0 +1,136 @@
+            )
+
+        result = future.result()
+        
+        #log_diagnostic_message(result)
+
+        if target_webhook:
+            result = asyncio.run_coroutine_threadsafe(
+                send_message_to_webhook(result, target_webhook, request_payload),
+                app.bot.loop
+            )
+            logging.info("Sent message to target webhook\n".format(result))
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)